image-builder/image_builder.sh

```diff
@@ -8,6 +8,8 @@ set -e
 
 [ -n "${DEBUG}" ] && set -x
 
+DOCKER_RUNTIME=${DOCKER_RUNTIME:-runc}
```

Comment:
what about naming it `CONTAINER_RUNTIME` as this will also be used for `podman`, right?

Reply:
yeah, but then I'll need to change it in `root.sh` too

https://github.com/kata-containers/osbuilder/blob/0e4c497c68fcd405fe5946a464274cc40d7cea9e/rootfs-builder/rootfs.sh#L21

Reply:
oh ok, if someone is already using `DOCKER_RUNTIME`, we may break them.